### PR TITLE
chore: Update close message error strings

### PR DESCRIPTION
**Pull request by timarney**
# Summary | Résumé

Updating strings.

### CODE REVIEW

File: i18n/translations/fr/form-builder.json [hunk 1]
```diff
@@ -1210,7 +1210,7 @@
         "text3": "[FR]For example 21:00"
```
Comment (timarney) [inline i18n/translations/fr/form-builder.json:1210]: @anikbrazeau here's the other one.

Comment (anikbrazeau) [inline i18n/translations/fr/form-builder.json:1210]:
```suggestion
        "text3": "Par exemple 21 h 00"
```

**Comment by timarney** [comment]
PR https://github.com/cds-snc/platform-forms-client/pull/4531

**Review by thiessenp-cds — APPROVED** [review]
LGTM :)

**Pull request merged**